docker/docker-compose.yml

```diff
@@ -74,7 +74,8 @@ services:
           - CHAIN
           - DEBUG
         volumes:
-            - shared:/rpc
+          - shared:/rpc
+          - lnd:/root/.lnd
```

Comment:
I just had a look at #697 which has the same goal.
Adding this persistent volume will create a problem for the tutorial in `README.md`.
Before this change, both nodes `alice` and `bob` stored their data in the non-persistent file system of their respective containers.
Now, they would both use the same volume and overwrite the data.
I don't really like like the approach in #697 that adds two extra configurations for `alice` and `bob` in the `docker-compose.yml`.
How about this instead (since users will copy/paste the commands from the tutorial anyway):
Changes to `docker/README.md`:

```
# Init bitcoin network env variable:
$ export NETWORK="simnet" 

# Create persistent volumes for alice and bob.
docker volume create simnet_lnd_alice
docker volume create simnet_lnd_bob

# Run the "Alice" container and log into it:
$ docker-compose run -d --name alice --volume simnet_lnd_alice:/root/.lnd lnd_btc
```

and further down:

```
# Run "Bob" node and log into it:
$ docker-compose run -d --name bob --volume simnet_lnd_bob:/root/.lnd lnd_btc
```

Reply:
good catch :fish: